### ISSUE TITLE
add auth token in header on swagger pages

### ISSUE THREAD

**Issue by JosefAssadERST** [TheRacetrack/racetrack#82]
In the swagger pages you get a very helpful text field with a "copy" button for the curl calls corresponding to the swagger example calls. Right now we have to manually figure out what the HTTP header for the token is, and fetch the token by hand from another page, and add it to that curl command.

Surely we can override or extend the swagger framework to inject that so the copied command can be invoked directly?

It's not a big deal but it's a fairly obvious and recurring paper cut.

**Comment by iszulcdeepsense** [comment]
Good reasoning.

Just want to note that in the current design, PUB is the authentication gateway and it's the last place where the auth token is present. So Fatman may not know who was calling and what token should be displayed. The token can't be rendered at page in advance since we don't know which user or ESC will be calling it, so it ought to be taken from the request

**Comment by JosefAssadERST** [comment]
Yeah I think we discussed before and it was hard for some reason and that's entirely fair. I wanted to note the paper cut though so in the worst case we're aware.
